kie-remote/kie-remote-common/src/main/java/org/kie/remote/common/rest/RestEasy960Util.java

```diff
@@ -55,14 +55,15 @@ public static Variant getVariant(HttpHeaders headers) {
                 fixedAcceptHeaders.add(header.replaceAll("q=\\.", "q=0.")); 
             }
             acceptHeaders = fixedAcceptHeaders;
-        }
```

Comment:
Good catch, @mswiderski. I didn't see the NPE until I looked at the [ServerDrivenNegotiation method](https://github.com/mswiderski/droolsjbpm-integration/blob/BZ-1319553/kie-remote/kie-remote-common/src/main/java/org/kie/remote/common/rest/variant/ServerDrivenNegotiation.java#L136).